### DISCUSSION TITLE
Clean database - start from scratch for orders

### DISCUSSION THREAD

**Discussion by slamer59** [saleor/saleor#11170] [General] [answered]
I want to start from a cleaned database for orders (in dev environement) how can I achieve this without removing products, etc...

How can I achieve this ?

**Comment by maarcingebala** [comment]
Hey, are you refering to open-source Saleor or a Saleor Cloud instance? When you setup local environment with open-source Saleor the database is empty by default. You can populate it with example products with the `populatedb` script. Do you mean keeping the products, but removing all orders?

**Comment by slamer59** [comment]
Hello,
I am using the OSS version.

Yes I have the products and I want to remove the order.




Le lun. 14 nov. 2022, 16:03, Marcin Gębala ***@***.***> a
écrit :

**Comment by maarcingebala** [comment, marked as answer]
You can do that by removing them from the database using Django shell. Run `python manage.py shell` and type:

```
from saleor.order.models import Order
from saleor.payment.models import Transaction, Payment

Transaction.objects.all().delete()
Payment.objects.all().delete()
Order.objects.all().delete()
```

It should remove all orders and payment objects, while keeping everything else.

**Comment by slamer59** [comment]
Oh ! Great idea !

Thanks. :)


Le lun. 14 nov. 2022, 18:12, Marcin Gębala ***@***.***> a
écrit :

**Comment by slamer59** [comment]
Hello,
Just for making a small correction. 
One need to delete **TransactionItem** before deleting Order

```
from saleor.order.models import Order
from saleor.payment.models import Transaction, TransactionItem, Payment

Transaction.objects.all().delete()
Payment.objects.all().delete()
TransactionItem.objects.all().delete()
Order.objects.all().delete()
```
